episodes/designing-a-light-microscopy-experiment.md

```diff
@@ -220,9 +220,9 @@ measure the nucleus 'roundness' (a measure of how circular it is). In 3D, we
 could measure nucleus 'sphericity' (a measure of how spherical it is). Which 
 measures you use will often depend on which image analysis software you use. 
 Many analysis packages have a wide range of shape (morphological) features 
-built-in e.g. Napari has a plugin called 'napari-skimage-regionprops' that 
-offers [many different features
-](https://www.napari-hub.org/plugins/napari-skimage-regionprops#features).
+built-in e.g. Napari has a plugin called 'napari-skimage', which provides 
+access to scikit-image's `regionprops` that offers [many different features
+](https://scikit-image.org/docs/stable/api/skimage.measure.html#skimage.measure.regionprops).
```

Comment:
Does this suggestion improve the accessibility of the link? In it's current form it's not immediatly clear what you're clicking on.
```suggestion
access to scikit-image's `regionprops`. See [scikit-image's documentation](https://scikit-image.org/docs/stable/api/skimage.measure.html#skimage.measure.regionprops "Documentation for regionprops") for details on the available features.